Create svx-1.3.1.md

anastasiaUsik opened the pull request.

anastasiaUsik committed:
Create svx-1.3.1.md

anastasiaUsik committed:
Update SUMMARY.md

derek-meeco commented:
Now version 1.3.2

> Team, SVX Platform 1.3.2 has been released. The only component which has changed is Portal, from version 2.21.0 to 2.22.0

javereec commented on releases/svx-1.3.1.md:
```diff
@@ -0,0 +1,48 @@
+
+### Portal
+
+* Created a new Toolbar.
```
@anastasiaUsik can you specify where you can find this Toolbar

javereec commented on releases/svx-1.3.1.md:
```diff
@@ -0,0 +1,48 @@
+### Portal
+
+* Created a new Toolbar.
+* Updated the screen View credential based on security aspects and whether a passphrase was entered or not.
```
What was the security aspect that was enhanced?

anastasiaUsik replied:
@javereec as far as I know Terry did some work, could you please help me with rewriting this point?
everything else has been updated

javereec reviewed (commented):
Do these release notes address 1.3.1 or 1.3.2.
Have a look here for paltform changelog: https://bitbucket.org/meeco/gateway/src/develop/PLATFORM.md

anastasiaUsik committed:
Update svx-1.3.1.md

anastasiaUsik committed:
Update svx-1.3.1.md

I think it's better to skip the explanation of 1.3.1 because the described items in the 1.3.0 release notes have been included in the 1.3.1 release, just to avoid future disorder @javereec

anastasiaUsik committed:
Rename svx-1.3.1.md to svx-1.3.2.md

anastasiaUsik committed:
Update SUMMARY.md
